src/azul/service/service_config/request_config.json

```diff
@@ -53,46 +53,46 @@
   },
   "manifest": {
     "bundles": {
-      "bundle_uuid": "uuid",
-      "bundle_version": "version"
+      "bundle.uuid": "uuid",
+      "bundle.version": "version"
     },
     "contents.files": {
-      "file_content_type": "content-type",
-      "file_name": "name",
-      "file_sha256": "sha256",
-      "file_size": "size",
-      "file_uuid": "uuid",
-      "file_version": "version",
-      "file_indexed": "indexed",
-      "file_format": "file_format"
+      "file.content_type": "content-type",
+      "file.name": "name",
+      "file.sha256": "sha256",
+      "file.size": "size",
+      "file.uuid": "uuid",
+      "file.version": "version",
+      "file.indexed": "indexed",
+      "file.format": "file_format"
     },
     "contents.cell_suspensions": {
-      "total_estimated_cells": "total_estimated_cells"
+      "cell_suspension.estimated_cell_count": "total_estimated_cells"
     },
     "contents.protocols": {
-      "instrument_manufacturer_model": "instrument_manufacturer_model",
-      "library_construction_approach": "library_construction_approach"
+      "protocol.instrument_manufacturer_model": "instrument_manufacturer_model",
+      "protocol.library_construction_method": "library_construction_approach"
     },
     "contents.projects": {
-      "document_id": "document_id",
-      "institutions": "institutions",
-      "laboratory": "laboratory",
-      "project_shortname": "project_shortname",
-      "project_title": "project_title"
+      "project.document_id": "document_id",
+      "project.institutions": "institutions",
+      "project.laboratory": "laboratory",
+      "project.project_short_name": "project_shortname",
+      "project.project_title": "project_title"
     },
     "contents.specimens": {
-      "biological_sex": "biological_sex",
-      "specimen_id": "biomaterial_id",
-      "specimen_document_id": "document_id",
-      "disease": "disease",
-      "donor_biomaterial_id": "donor_biomaterial_id",
-      "donor_document_id": "donor_document_id",
-      "genus_species": "genus_species",
-      "organ": "organ",
-      "organ_part": "organ_part",
-      "organism_age": "organism_age",
-      "organism_age_unit": "organism_age_unit",
-      "preservation_method": "preservation_method"
+      "specimen.sex": "biological_sex",
```

Comment:
Specimens don't have sex. Donors do. Please double check the mapping.

Reply:
Does this mean the correct column name for `biological_sex` should be `donor.sex`? Should `donor_biomaterial_id ` be `donor.biomaterial_id` and `donor_document_id ` be `donor.document_id`?

Reply:
`donor_organism.biomaterial_core.biomaterial_id`
`specimen_from_organism.biomaterial_core.biomaterial_id`